setup.php

```diff
@@ -24,22 +24,22 @@
 <body>
 <h1>Setup Selfauth</h1>
 <div class="instructions">In order to configure Selfauth, you need to fill in a few values, this page helps generate those options.</div>
-<?php if(isset($_POST['username'])):?>
+<?php if (isset($_POST['username'])) :?>
```

Comment:
This is the only change that frustrates me: a space between `)` and `:` but not between `:` and `?>`.